app/modules/core/error_handler.py

```diff
@@ -34,7 +34,7 @@ async def on_command_error(self, ctx: Context, error: CommandError):
         """Commands error handler."""
         
         if ErrorHandler.is_unknown_error(error):
-            logger.error(error)
+            logger.exception(error)
```

Comment:
Why the change from error to exception?

Reply:
`exception()` prints the stacktrace too.